workflow/scripts/neighborhoods.py

```diff
@@ -446,21 +447,16 @@ def count_bam(
 
 def count_tagalign(tagalign, bed_file, output, genome_sizes):
     index_file = tagalign + ".tbi"
-    if os.path.exists(index_file):
-        command1 = ""
-    else:
-        command1 = "tabix -p bed {tagalign} | cut -f1-3".format(**locals())
+    if not os.path.exists(index_file):
+        cmd = f"tabix -p bed {tagalign} | cut -f1-3"
+        check_call(cmd, shell=True)
 
-    command2 = 'bedtools coverage -counts -b {tagalign} -a {bed_file} | awk \'{{print $1 "\\t" $2 "\\t" $3 "\\t" $NF}}\' '.format(
-        **locals()
-    )
+    cmd1 = f"bedtools coverage -counts -sorted -b {tagalign} -a {bed_file}"
```

Comment:
separate out the piped commands because you won't catch errors. bedtools coverage was constantly OOMing, but an error wouldn't get raised. Now it does

Reply:
Also add sorted option for memory improvement